readme.txt

```diff
@@ -116,6 +116,10 @@ Please visit the [Sensei Blog](https://senseilms.com/blog/) or sign up for our [
 
 == Changelog ==
 
+2022-10-27 - Version 4.8.0
```

Comment:
I think now that we added this one, we should remove the "2022-10-11 - version 4.7.0", keeping only the 3 last versions.

Reply:
Fixed in [6cda490](https://github.com/Automattic/sensei/pull/6011/commits/6cda4900fe0b0edb95b86f6733f74f32fbaa7cb7)